elide-datastore/elide-datastore-hibernate5/src/main/java/com/yahoo/elide/datastores/hibernate5/HibernateTransaction.java

```diff
@@ -180,7 +180,7 @@ public void commit() {
 
         final Pagination pagination = filterScope.hasPagination()
                 ? filterScope.getRequestScope().getPagination()
-                : null;
+                : Pagination.getDefaultPagination();
```

Comment:
We need to be careful with loads.  If there was a filter join, pagination won't work and we'd be better off passing null (in the cases where there is no pagination defined at all).